cli/src/utils.rs

```diff
@@ -227,7 +227,7 @@ pub fn print_receipt(receipt: &TransactionReceipt) {
     };
 
     println!(
-        "\n#####\n{}Hash: 0x{}{}\nBlock: {}\n{}\n",
+        "\n##### {chain}\n{}Hash: 0x{}{}\nBlock: {}\n{}\n",
```

Comment:
use consistent pattern

Reply:
should we log transactions separated by chain?

Reply:
yes definitely, it's part of the `stdout` follow-up i mentioned